marv/app/listing.py

```diff
@@ -251,6 +257,12 @@ def fileset_listing():
 
         site = current_app.site
         query = Listing.query
+        if not filters:
+            #Only show recent stuff if no filter is given
+            time = datetime.today() - timedelta(days=2)
+            date = time.strftime("%Y-%m-%dT00:00:00-04:00");
```

Comment:
The filter you create in the front end has that -4 because that is your current time zone... so now you can end up with less or more than 48 hours if the server time zone is not at -4. Removing time offset and using date time in utc is preferred I think.